.travis.yml

```diff
@@ -3,6 +3,7 @@ elixir:
   - 1.2.3
 otp_release:
   - 18.2
+  - 19.0
```

Comment:
I would remove this right now and add it back in once a new update to `excoveralls` comes out with 19 support.